packages/core/src/shared/telemetry/vscodeTelemetry.json

```diff
@@ -17,6 +17,17 @@
             "allowedValues": ["Create", "Update"],
             "description": "SSM Publish Document operation type"
         },
+        {
+            "name": "stepFunctionsSyncType",
+            "type": "string",
+            "allowedValues": ["manualSaveWFS", "manualSaveVSCode", "autoSyncWFS"],
+            "description": "The action type that triggered the save or sync event between SFN Workflow Studio and the local source file."
+        },
+        {
+            "name": "isInvalidJson",
+            "type": "boolean",
+            "description": "Indicates whether the message contains an invalid JSON definition."
```

Comment:
please read https://github.com/aws/aws-toolkit-vscode/blob/master/docs/telemetry.md

this looks like a failure mode. throw an error from your code, and then `telemetry.xx.run()` will correctly set `result=Failed` and `reason=<error code>`. Don't add a highly-specific boolean to signal a failure.

Reply:
This is not a failure, but rather an expected case for which we are syncing webview content with the file as usual. This variable is just to track this case in telemetry, not to handle this differently

Reply:
How is invalid json not a failure?

Reply:
Since we are launching our custom editor by default, we want to sync the webview content with local file as much as we can. If the user opens code editor inside our integration and types something making JSON invalid, we still want to sync that invalid JSON to the local file as a string to prevent user from losing any data (e.g. if they close VSCode)